docs/server/source/dev-and-test/setup-bdb-host.md

```diff
@@ -0,0 +1,56 @@
+# Set Up BigchainDB Node on Local Dev Machine
```

Comment:
can be please add expected configurations of the local dev machine here? e.g OS and recommended spec? does this documentation apply to all dev environments like MacOS, ubuntu etc?

Reply:
The requirements are mentioned in the previous document which links to this one. `Set Up & Run a Dev/Test Node`